examples/README.md

```diff
@@ -2082,9 +2082,7 @@ Bidding is done via an online application and bids are received as events are as
           "name": "StoreCarAuctionBid",
           "type": "event",
           "start": {
-              "schedule": {
-                "interval": "2020-03-20T09:00:00Z/2020-03-20T15:00:00Z"
-              }
+              "schedule": "2020-03-20T09:00:00Z/2020-03-20T15:00:00Z"
```

Comment:
This is confusing because the field is "schedule" but that's not a schedule, it's a time interval...

Reply:
it can also be a time repeating interval. would it make more sense to use that for the examples?

Reply:
Yes, that would probably make more sense for the examples.  Is there a way to restrict that it can only be a repeating interval?  Supporting a time range like the above seems to cause a lot of confusion on what that would mean.

Reply:
+1
Good point. I think also we should restrict in text.